docs/source/daml/reference/templates.rst

```diff
@@ -184,3 +184,24 @@ Contract Keys and Maintainers
 
   Because of this, the ``key`` must include the ``maintainer`` ``Party`` or parties (for example, as part of a tuple or record), and the ``maintainer`` must be a signatory.
 - For a full explanation, see :doc:`/daml/reference/contract-keys`.
+
+Interface Instances
+*******************
+
+.. warning::
+  This feature is under active development and not officially supported in
+  production environments.
```

Comment:
I don't think this warning needs to be here, since the feature is currently opt-in and by the time the release is out interface instances are going to be supported.

Reply:
I think it's better to remove it when the release actually happens, no?

Reply:
The only way in which a person might see this is by explicitly going to the snapshot documentation (which is not listed on the docs site and needs to be accessed explicitly). I think that the risk of people reading snapshot documentation _and_ use the 2.4.0 version _and_ activate interfaces explicitly _and_ go in production _and_ claim it happened by mistake is low, but if you think there's an actual risk I'd recommend to open a ticket to track this and put it immediately on the [release planning document](https://docs.google.com/document/d/1FaBFuYweYt0hx6fVg9rhufCtDNPiATXu5zwN2NWp2s4/edit#heading=h.ao0anu1yg4w4) as a blocker for 2.5.0 (I think it's fair to say that it's not properly blocking, but the effort to remove it is so low that I would put it there just to make sure we don't forget about it).

Reply:
Everything else is fine so changed my review to green, I appreciate the caution as long as we keep track effectively of this. 🙂

Reply:
I've removed the warning here and the one at the top of Reference: Interfaces. The warning about required interfaces stays.